mos-platform/rpc8e/librpc8e/include/rpc8e.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 Adrian "asie" Siekierka
+ *
+ * This software is provided 'as-is', without any express or implied
+ * warranty. In no event will the authors be held liable for any damages
+ * arising from the use of this software.
+ *
+ * Permission is granted to anyone to use this software for any purpose,
+ * including commercial applications, and to alter it and redistribute it
+ * freely, subject to the following restrictions:
+ *
+ * 1. The origin of this software must not be misrepresented; you must not
+ *    claim that you wrote the original software. If you use this software
+ *    in a product, an acknowledgment in the product documentation would be
+ *    appreciated but is not required.
+ *
+ * 2. Altered source versions must be plainly marked as such, and must not be
+ *    misrepresented as being the original software.
+ *
+ * 3. This notice may not be removed or altered from any source distribution.
+ */
+
+#pragma once
```

Comment:
Prefer old-fashioned include guards; `#pragma once` ties in a poorly-specified, compiler-specific notion of file identity that is more difficult to reason about than a textual identifier. (Soft and hard links, bind mounting, etc make "is this the same file" a complex question).

Reply:
Done.